### PR TITLE
Add custom theme Support

### PR DESCRIPTION
**Pull request by acellam**

### CODE REVIEW

File: src/ui/_switch-theme.scss [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        }
+    }
+}
+
+@mixin android {
+    .widget-switch-btn-wrapper {
+        &.checked {
+            &.widget-switch-btn-wrapper-default {
+                @include bootstrap-style-android($default-android-color);
+                .widget-switch-btn {
```
Comment (edwinmp) [inline src/ui/_switch-theme.scss]: Add white space above.

File: src/ui/_switch-theme.scss [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+                    background: $default-android-color;
+                }
+            }
+
+            &.widget-switch-btn-wrapper-success {
+                @include bootstrap-style-android($brand-success);
+                .widget-switch-btn {
```
Comment (edwinmp) [inline src/ui/_switch-theme.scss]: Add white space above.

**Review by edwinmp — CHANGES_REQUESTED** [review]
Awesome... Thanks for the assist.

File: src/ui/_switch-theme.scss (hunk 1 above)
Comment (acellam) [reply src/ui/_switch-theme.scss]: Done

File: src/ui/_switch-theme.scss (hunk 2 above)
Comment (acellam) [reply src/ui/_switch-theme.scss]: Done

**Pull request merged**